frontend/src/lib/components/icons.stories.tsx

```diff
@@ -19,6 +19,7 @@ export default {
     title: 'Lemon UI/Icons',
     parameters: {
         options: { showPanel: false },
+        chromatic: { disableSnapshot: true }, // There are too many icons, the snapshots get very big in table form
```

Comment:
I think the current approach is very valid for components, it's just that icons aren't really components. _Maybe_ this would work if each icon had its own snapshot, but icons never really change anyway, so this doesn't seem important at the moment.